docs/storage/prometheus.md

```diff
@@ -50,6 +50,8 @@ Metric name | Type | Description | Unit (where applicable) | option parameter |
 `container_hugetlb_failcnt` | Counter | Number of hugepage usage hits limits | | hugetlb |
 `container_hugetlb_max_usage_bytes` | Gauge | Maximum hugepage usages recorded | bytes | hugetlb |
 `container_hugetlb_usage_bytes` | Gauge | Current hugepage usage | bytes | hugetlb |
+`container_misc_usage` | Gauge | Current usage of the resource | | misc |
+`container_misc_events` | Counter | Number of times the usage for the resource was about to go over the max boundary | | misc |
```

Comment:
`resource` is IMHO too generic term.  Maybe "misc cgroup HW resource"?

PS. Same thing goes for the Prometheus metrics descriptions.

Alternatively "resource" could be a link to: https://www.kernel.org/doc/html/next/admin-guide/cgroup-v2.html#misc

Reply:
updated to "misc scalar resource specified by the label"